.github/workflows/checkpatch.yml

```diff
@@ -0,0 +1,19 @@
+on: push
+
+name: Check Code Style
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+    env:
+      DL_DIR: ../downloads
+      BUILD_DIR: ../build
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Run checkpatch
+        run: |
+            git diff -U20 HEAD~40 | \
+                    filterdiff -x "a/src/jtag/drivers/libjaylink/*" -x "a/tools/git2cl/*" \
+                    -x "b/src/gnulib/*" |
```

Comment:
It seems this part can be dropped as `src/gnulib` is no longer present.

Reply:
Good point.